encryptor/queryDataEncryptor.go

```diff
@@ -107,8 +108,11 @@ func UpdateExpressionValue(expr sqlparser.Expr, coder DBDataCoder, updateFunc fu
 		case sqlparser.StrVal, sqlparser.HexVal, sqlparser.PgEscapeString:
 			rawData, err := coder.Decode(val)
 			if err != nil {
-				if err != errUnsupportedExpression {
-					logrus.WithField(logging.FieldKeyEventCode, logging.EventCodeErrorCodingCantDecodeSQLValue).WithError(err).Errorln("Can't decode data")
+				if err == utils.ErrDecodeEscapedString || err == errUnsupportedExpression {
+					logrus.WithField(logging.FieldKeyEventCode, logging.EventCodeErrorCodingCantDecodeSQLValue).
+						WithError(err).
+						Warningln("Can't decode data with unsupported coding format or unsupported expression")
```

Comment:
Shall we write anywhere that this is an error while encrypting value in Transparent mode? Because users will get plaintext data due to error, but will see "decoding" error, which is misleading.

Maybe add extra log on line 146/150 "Encryption error"

https://github.com/cossacklabs/acra/pull/314/files#diff-4aa4720957d4bb4bcaf1c7b968531a31R150

Reply:
it will be logged on a higher level
https://github.com/cossacklabs/acra/blob/47c33ac882700ee750bd3033827936dc40921f7e/encryptor/queryDataEncryptor.go#L213

Reply:
got it, thank you! great.